client/reader/post-options/index.jsx

```diff
@@ -87,6 +95,27 @@ var PostOptions = React.createClass( {
 		window.open( 'https://wordpress.com/abuse/?report_url=' + encodeURIComponent( this.props.post.URL ), '_blank' );
 	},
 
+	getFollowUrl() {
+		return this.state && this.state.feed ? this.state.feed.get( 'feed_URL' ) : this.props.post.site_URL;
+	},
+
+	getFeed() {
+		const feedId = get( this.props, 'post.feed_ID' );
+		if ( ! feedId || feedId < 1 ) {
+			return;
+		}
+
+		const feed = FeedStore.get( feedId );
+
+		if ( ! feed ) {
+			setTimeout( function() {
```

Comment:
I used this to get around the 'dispatch within a dispatch' problem, and I noticed that we've used it before:

https://github.com/Automattic/calypso-pre-oss/commit/666a1b569859765c6e127d4e222143541f002900

Is there a cleaner way to work around this?